lib/src/components/NxToast/NxToast.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { useState, useContext, useEffect, useRef } from 'react';
+import classnames from 'classnames';
+import NxToastContainerContext from './contexts';
+
+import { nxToastPropTypes, NxToastProps } from './types';
+
+const NxToast = (props: NxToastProps) => {
+  const { onClose, children } = props;
```

Comment:
`toastId` isn't actually used here - and it shouldn't be.  It's up to the calling code how it wants to distinguish between one toasts `onClose` and another, and using closure scope as your examples do is one valid way.  That requires no special support from `NxToast`.  So you should remove `toastId` from the `Props` and the docs.

Reply:
> `toastId` isn't actually used here - and it shouldn't be. It's up to the calling code how it wants to distinguish between one toasts `onClose` and another, and using closure scope as your examples do is one valid way. That requires no special support from `NxToast`. So you should remove `toastId` from the `Props` and the docs.

Absolutely. It's removed here: [bba523c](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/bba523c7b55c4033284813bb082371eb38ba9d08)